### PR TITLE
feat(electron): add dl count and version

### PR DESCRIPTION
**Pull request by mrchops1024**
Add download count as a column to the get addons page.
Add addon version below the addon name.

Implements #69 and #141 for v2.x.

### CODE REVIEW

**Comment by BloodDragon2580** [comment]
That will be nice

File: wowup-electron/src/app/addon-providers/curse-addon-provider.ts [hunk 1]
```diff
@@ -24,6 +24,7 @@ import { CurseFile } from "common/curse/curse-file";
 import { CurseReleaseType } from "common/curse/curse-release-type";
 import { CurseGetFeaturedResponse } from "app/models/curse/curse-get-featured-response";
 import * as CircuitBreaker from "opossum";
+import { WowUpService } from "app/services/wowup/wowup.service";
```
Comment (jliddev) [inline wowup-electron/src/app/addon-providers/curse-addon-provider.ts:27]: Can be removed

**Review by jliddev — COMMENTED** [review]
Can we add a subscriber in the Get Addons page to update the list if I go to options and change the default install channel?

**Comment by mrchops1024** [comment]
> Can we add a subscriber in the Get Addons page to update the list if I go to options and change the default install channel?

Done.

**Pull request merged**